### PR TITLE
Feature/deploy heroku

### PR DESCRIPTION
**Pull request by andrewvallejo**
Setup and deploy Heroku

## Description
Heroku seamlessly intergrates with Postgres and Express API.... Although it was not a seamless experience. Things to remember: Always return all response calls and follow this [tutorial](https://www.taniarascia.com/node-express-postgresql-heroku/). Connect Hiroku first if possible. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactor

## Motivation and Context
To deploy our backend and complete our stretch goal for our project.

## Concerns
- Suspicious if I ever have to never with repo again!

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
+// module.exports = pool
 
-const connectionString = `postgresql://${process.env.DB_USER}:${process.env.DB_PASSWORD}@${process.env.DB_HOST}:${process.env.DB_PORT}/${process.env.DB_DATABASE}`
+// require('dotenv').config()
+
+const {Pool} = require('pg')
 
 const pool = new Pool({
-  connectionString: isProduction ? process.env.DATABASE_URL : connectionString
+  user: "od_admin",
```
Comment (rachaelcarroll) [inline config.js:20]: Don't upload this file with your password

File: database.sql [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+('On a <script> tag, what are async and defer attributes?'),
+('What is the difference between == and ===?'),
+('How can you share code between files?'),
+('What is the the difference between synchronous and asynchronous functions?'),
+('What tools and techniques do you use debugging JavaScript code?'),
+('What language constructions do you use for iterating over object properties and array items?'),
+('How do you organize your code? (module pattern, classical inheritance?)'),
+('What was the most interesting thing you learned about coding at Turing?'),
+('How do you go about deploying a project with GH pages, Heroku, or any other service?'),
+('How does the web work?');
```
Comment (rachaelcarroll) [inline database.sql:60]: Nice seeding of the database with these questions. Do we need to update this as we add more to the database ?

File: server.js [hunk 3]
```diff
@@ -1,6 +1,11 @@
 const express = require("express")
 const cors = require("cors");
-pool = require( "./config");
+const {Pool} = require('pg')
+
+const pool = new Pool({
+  connectionString: "postgres://afuxtawmyieyjo:13bb8d5ece25a9242c5ca5f55001027d17f329cfd3a91cecf780c5024e0239a4@ec2-3-237-55-96.compute-1.amazonaws.com:5432/d9aqkclakvsmeg",
```
Comment (rachaelcarroll) [inline server.js:6]: Nice job finding this connection string from our heroku account

File: config.js (hunk 1 above)
Comment (andrewvallejo) [reply config.js:20]: The more you learn! Happy I didnt use my personal password... dont check tho!

File: database.sql (hunk 2 above)
Comment (andrewvallejo) [reply database.sql:60]: Unsure.

**Pull request merged**